File: app/src/test/java/medlife/com/br/model/OrderStatusTest.java
package medlife.com.br.model;

import static org.junit.Assert.*;
import org.junit.Test;

public class OrderStatusTest {
    @Test
    public void testEnumValues() {
        assertEquals(OrderStatus.ORDER_CONFIRMED, OrderStatus.valueOf("ORDER_CONFIRMED"));
        assertEquals("ORDER_CONFIRMED", OrderStatus.ORDER_CONFIRMED.name());
    }
} 

File: app/src/test/java/medlife/com/br/model/OrderStatusTest.kt
package medlife.com.br.model

import org.junit.Assert.assertEquals
import org.junit.Test

class OrderStatusTest {
    @Test
    fun testEnumValues() {
        assertEquals(OrderStatus.ORDER_CONFIRMED, OrderStatus.valueOf("ORDER_CONFIRMED"))
        assertEquals("ORDER_CONFIRMED", OrderStatus.ORDER_CONFIRMED.name)
    }
}
